Homeworks/HW8/UI.py

```diff
@@ -0,0 +1,25 @@
+import fileIO
+import view
+
+def menu(data):
+    while True:
+        answer = view.show_menu()
+        if answer == 1:
+            view.show_data(data)
+        elif answer == 2:
+            # fileIO.add_data(data)
+            str_data = input('Enter your data delimited by tab> ')
+            row = str_data.split('\t')
+            data.append([row[0], row[1], row[2]])
```

Comment:
```suggestion
            data.append(row)
```
Не понимаю, зачем вы берете конкретный элемент из списка по индексу, потом помещаете их же в список.